transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/Defaults.java

```diff
@@ -82,12 +79,16 @@ private static Properties loadDefaultVersions() {
           HiveWrapperGenerator.class,
           JavaLanguageVersion.of(8),
           ImmutableList.of(
-              getDependencyConfiguration(IMPLEMENTATION, "com.linkedin.transport:transportable-udfs-hive", "transport"),
-              getDependencyConfiguration(COMPILE_ONLY, "org.apache.hive:hive-exec", "hive")
+              DependencyConfiguration.of(IMPLEMENTATION, "com.linkedin.transport:transportable-udfs-hive", getVersion("transport")),
+              DependencyConfiguration.builder()
+                  .configurationType(COMPILE_ONLY)
+                  .module("org.apache.hive:hive-exec")
+                  .version(getVersion("hive"))
+                  .exclude(PackageModule.of("org.apache.calcite"))
```

Comment:
How about `exclude(ImmutableList.of(ImmutableMap.of("group", "org.apache.calcite"))` and remove PackageModule altogether?

Reply:
will follow